src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp

```diff
@@ -248,6 +248,18 @@ namespace {
         }
     }
 
+    void Schedule::handleDRSDTCON(const HandlerContext& handlerContext, const ParseContext&, ErrorGuard&) {
+        std::size_t numPvtRegions = this->m_static.m_runspec.tabdims().getNumPVTTables();
+        std::vector<double> max(numPvtRegions);
+        std::vector<std::string> options(numPvtRegions);
+        for (const auto& record : handlerContext.keyword) {
+            std::fill(max.begin(), max.end(), record.getItem("DRSDT_MAX").getSIDouble(0));
```

Comment:
Use the compiled constants:
```C++
const auto& item = record.getItem<ParserKeywords::DRSDTCON::DRSDT_MAX>().getSIDouble(0);
```

Reply:
Ok, I can update the rest of the file accordingly to make it consistent.